styleguide/source/_patterns/03-templates/04-topic/topic.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
That you added the blank .json file here is a good thing IMO. Hopefully it will prompt us to fill it out later. :P 

In general, maybe we should establish that Templates get .json files as a rule?